### PR TITLE
HPCC-12063 Document eclcc --keywords option

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@jamienoss Please review

### CODE REVIEW

File: docs/HPCCClientTools/CT_Mods/ECLCC.xml [hunk 1]
```diff
@@ -446,6 +446,12 @@
               </row>
 
               <row>
+                <entry>--keywords</entry>
+
+                <entry>Outputs a list of ECL reserved words to stdout</entry>
```
Comment (jamienoss) [inline docs/HPCCClientTools/CT_Mods/ECLCC.xml]: We should specify that this will in xml format.

Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/ECLCC.xml]: @jamienoss Ok done. Please have another look

File: docs/HPCCClientTools/CT_Mods/ECLCC.xml [hunk 2]
```diff
@@ -446,6 +446,13 @@
               </row>
 
               <row>
+                <entry>--keywords</entry>
+
+                <entry>Outputs a list of ECL reserved words to stdout (XML
```
Comment (jamienoss) [inline docs/HPCCClientTools/CT_Mods/ECLCC.xml]: @JamesDeFabia  Sorry for being pedantic and not catching this before but that should say "Outputs the lists..."

Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/ECLCC.xml]: @jamienoss Edits made, please have another look

**Comment by jamienoss** [comment]
@richardkchapman this is good to merge.

**Pull request merged**